packages/components/card/src/asset-card/AssetCard.mdx

```diff
@@ -0,0 +1,147 @@
+v4Doc: true
+---
+
+import { Heading, Stack, TextLink } from '@contentful/f36-components';
+import { ExternalLinkIcon } from '@contentful/f36-icons';
+import { Props } from '@contentful/f36-docs-utils';
+
+The AssetCard component is built on top of the [Card component](/components/card) and it provides more props
+to help you manage the assets of your Contentful implementation.
+
+## Table of contents
+
+- [Import](#import)
+- [Examples](#examples)
+  - [Basic usage](#basic-usage)
+  - [Card actions](#card-actions)
+  - [Loading state](#loading-state)
+  - [Different sizes](#different-sizes)
+- [Props](#props-api-reference)
+- [Content guidelines](#content-guidelines)
+- [Accessibility](#accessibility)
+- [Help improve this page](#help-improve-this-page)
+
+## Import
+
+```jsx static=true
+import { AssetCard } from '@contentful/f36-components';
+// or
+import { AssetCard } from '@contentful/f36-card';
+```
+
+## Examples
+
+The main props to show the content of your asset are `title` and `src`.
+They both accept strings as values and `src` needs the url of the asset’s location.
+To show a badge with the status of the entry,
+you can pass one of "archived", "changed", "deleted", "draft", "new", or "published" to the `status` prop.
+
+It’s also possible to pass the Asset’s type to the `type` prop. The allowed values are:
+
+- archive
+- audio
+- code
+- image
+- markup
+- pdf
+- plaintext
+- presentation
+- richtext
+- spreadsheet
+- video
+
+### Basic usage
+
+```jsx
+<AssetCard
+  status="published"
+  type="image"
+  title="Everest"
+  src="https://bit.ly/31yL3Ps"
+/>
+```
+
+### Card actions
+
+To show a `...` button with a menu in the card, pass an array of `MenuItem` components
+
+```jsx
+<AssetCard
+  status="published"
+  type="image"
+  title="Everest"
+  src="https://bit.ly/31yL3Ps"
+  actions={[
+    <MenuItem key="copy" onClick={() => alert('copy')}>
+      Copy
+    </MenuItem>,
+    <MenuItem key="delete" onClick={() => alert('delete')}>
+      Delete
+    </MenuItem>,
+  ]}
+/>
+```
+
+### Loading state
+
+```jsx
+<AssetCard
+  status="published"
+  type="image"
+  title="Everest"
+  src="https://bit.ly/31yL3Ps"
+  isLoading
+/>
+```
+
+### Different sizes
+
+```jsx
+<Stack>
+  <AssetCard
+    status="published"
+    type="image"
+    title="Everest"
+    src="https://bit.ly/31yL3Ps"
+    size="small"
+  />
+  <AssetCard
+    status="published"
+    type="image"
+    title="Everest"
+    src="https://bit.ly/31yL3Ps"
+    size="default"
+  />
+</Stack>
+```
+
+<Stack justifyContent="space-between" marginTop="spacing2Xl" marginBottom="spacingL">
+  <Heading id="props-api-reference" as="h2" marginTop="none" marginBottom="none">
+    Props (API reference)
+  </Heading>
+
+  <TextLink
+    href="https://v4-f36-storybook.netlify.app/?path=/story/components-card-assetcard"
+    target="_blank"
+    alignIcon="end"
+    icon={<ExternalLinkIcon />}
+  >
+    Open in Storybook
+  </TextLink>
+</Stack>
+
+<Props of="AssetCard" />
```

Comment:
```suggestion

## Props

<Props of="AssetCard" />
```

Reply:
should props have title to make the table of contents work?

Reply:
They have, check line 125

Reply:
This way of doing the title for the props list was introduced in this PR
https://github.com/contentful/forma-36/pull/1579

It's the only way to have a title and a button for storybook aligned